src/Illuminate/Foundation/Bootstrap/DetectEnvironment.php

```diff
@@ -24,4 +26,19 @@ public function bootstrap(Application $app)
             }
         }
     }
+
+    /**
+     * Detect if a custom env file matching the APP_ENV exists.
+     *
+     * @param  Illuminate\Contracts\Foundation\Application  $app
```

Comment:
There should be backslash: `@param  \Illuminate\Contracts\Foundation\Application  $app`